docs/bastion.md

```diff
@@ -0,0 +1,51 @@
+# Bastion in Kops
+
+Bastion provide an external facing point of entry into a network containing private network instances. This host can provide a single point of fortification or audit and can be started and stopped to enable or disable inbound SSH communication from the Internet, some call bastion as the "jump server".
+
+Note: Bastion will get setup for the cluster(by default) only when `--topology="private"`.
+
+* [More information on bastion from aws](http://docs.aws.amazon.com/quickstart/latest/linux-bastion/architecture.html)
+* [More information on bastion from gce](https://cloud.google.com/solutions/connecting-securely#bastion)
+
+## AWS
+
+### Specify instance type of bastion
+
+Instance types in AWS comprise varying combinations of CPU, memory, storage, and networking capacity and give you the flexibility to choose the appropriate mix of resources for your applications.
+```
+kops create cluster --bastion-instance-type="t2.large"
```

Comment:
Instead of yet another command line switch, can we make it a yaml / edit option?